GraphAlgorithms/LargestCliqueNaiveFinder.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphAlgorithms
+{
+    public static class LargestCliqueNaiveFinder
+    {
+        public static List<int> Find(Graph graph)
+        {
+            List<int> largestClique = new();
+            Find(graph, graph.Size, new List<int>(), 0, ref largestClique);
+            return largestClique;
+        }
+
+        private static void Find(Graph graph, int n, List<int> clique, int v, ref List<int> largestClique)
+        {
+            for (int i = v; i < n; i++)
+            {
+                clique.Add(i);
+                if (IsClique(graph, clique) && clique.Count > largestClique.Count)
+                    largestClique = new List<int>(clique);
+                Find(graph, n, clique, i + 1, ref largestClique);
+                clique.Remove(i);
```

Comment:
can we change `clique.Remove(i)` to `clique.RemoveAt(clique.Count - 1)`? The reason is that the first method has to do linear search for `i`, and the second one is O(1) when removing from the end